reconcile/utils/ocm/ocm.py

```diff
@@ -340,13 +351,42 @@ def create_cluster(ocm: OCM, name: str, cluster: OCMSpec, dry_run: bool):
                 return
         ocm._post(api, ocm_spec, params)
 
+    @staticmethod
+    def _scale_default_node_pool(ocm: OCM, cluster_name: str, node_count: int):
+        cluster_id = ocm.cluster_ids.get(cluster_name)
+        # todo get node pools from ocm with name prefix workers
+        node_pools = ocm.get_node_pools(cluster_id)
+        pools_to_scale = []
+        for pool in node_pools:
+            if pool["id"].startswith(OCMProductRosa.DEFAULT_NODE_POOL):
```

Comment:
Todo: add a validation to qontract-schemas, that prevents node pools from being created with prefix "workers"